pkg/volume/csi/csi_mounter_test.go

```diff
@@ -141,7 +141,8 @@ func TestMounterSetUp(t *testing.T) {
 	}
 
 	// Mounter.SetUp()
-	if err := csiMounter.SetUp(nil); err != nil {
+	fsGroup := int64(2000)
```

Comment:
Is this just a random number or does it mean something in particular? Maybe it should be a `const` variable

Reply:
@davidz627 this is a typical value that represents sine groupID.  It's only here to force the test path and is used nowhere else.